terraform/spec/dependabot/terraform/requirements_updater_spec.rb

```diff
@@ -76,6 +76,30 @@
         end
       end
 
+      context "and a <= requirement was previously specified" do
+        context "that is satisfied" do
+          let(:requirement) { "<= #{latest_version}" }
+          it { is_expected.to eq(requirements.first) }
+        end
+
+        context "that is not satisfied" do
+          let(:requirement) { "<= 0.1.9" }
+          its([:requirement]) { is_expected.to eq("<= #{latest_version}") }
+
+          context "specifying two digits" do
+            let(:requirement) { "<= #{latest_version.to_s[0, 3]}" }
```

Comment:
Personally, I would not use `#{latest_version}` in these tests, as it makes the test harder to read. Especially when you strip some string, it is not directly clear what is going on. So I suggest to explicitly writing out the requirements.